app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_status_transition_validator.py

```diff
@@ -15,21 +15,21 @@ def setUp(self):
         self.user = UserFactory()
 
     def test_update_experiment_status_error(self):
-        experiment = NimbusExperimentFactory.create(
-            status=NimbusExperiment.Status.DRAFT,
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.DRAFT_CREATED
         )
         serializer = NimbusExperimentSerializer(
             experiment,
             data={
-                "status": NimbusExperiment.Status.LIVE,
+                "status": NimbusExperiment.Status.COMPLETE,
                 "changelog_message": "test changelog message",
             },
             context={"user": self.user},
         )
         self.assertFalse(serializer.is_valid())
         self.assertEqual(
             serializer.errors["status"][0],
-            "Nimbus Experiment status cannot transition from Draft to Live.",
```

Comment:
I think we should keep the test for draft to live not valid transition

Reply:
This was a weird test, because we **can** go from Draft to Live (see [Publish Approve/Approve](https://github.com/mozilla/experimenter/blob/main/app/experimenter/docs/README.md#publish-approveapprove) workflow). I'm not sure why this test was here in the first place 🤷‍♀️

Reply:
This might be a "wait for Jared because there might be a reason we're missing" case

Reply:
Ahhh this is making sense now. This test is because this status validator is only checking what the UI is able to do, and it is not allowed to change draft to live